src/rich-text-container.component.js

```diff
@@ -59,6 +59,7 @@ export function RichTextContainer(props) {
   return (
     <RichTextContext.Provider value={contextValue}>
       {props.children}
+      <div style={{display: 'none'}}></div>
```

Comment:
Why below `props.children` instead of above? Also, this is related to the Edge bug, right?